gcp/table_gcp_storage_bucket_object.go

```diff
@@ -0,0 +1,349 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/storage/v1"
+)
+
+func tableGcpBucketStorageObject(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_storage_bucket_object",
```

Comment:
```suggestion
		Name:        "gcp_storage_object",
```

Even though TF calls it `gcp_storage_bucket_object`, I think we should deviate here, unless this new name is ambiguous and there are other GCP Storage objects